corehq/blobs/management/commands/run_blob_import.py

```diff
@@ -44,8 +50,10 @@ def worker(filename, worker_number):
     # ^^^ Yeah, not great. Maybe filesystem caching helps.
     blob_db = get_blob_db()
     with tarfile.open(filename, 'r:gz') as tgzfile:
-        for i, tarinfo in enumerate(tgzfile):
-            if i % NUM_WORKERS == worker_number:
-                key = tarinfo.name
-                fileobj = tgzfile.extractfile(tarinfo)
-                blob_db.copy_blob(fileobj, key)
+        iterable = (tarinfo for i, tarinfo in enumerate(tgzfile)
+                    if i % NUM_WORKERS == worker_number)
+        prefix = f"Worker {worker_number}"
+        for tarinfo in with_progress_bar(iterable, length, prefix=prefix, oneline=False):
```

Comment:
The `oneline` param here puts each update on a new line, rather than overwriting in place.  This way the parallel tasks won't be overwriting each other's output, though it does make it much more spammy.